extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.bindings.js

```diff
@@ -0,0 +1,82 @@
+angular.module('PaperUI.controllers.configuration.bindings', [ 'ngRoute' ]).config([ '$routeProvider', function($routeProvider) {
+    // Configure the routes for this controller
+    $routeProvider.when('/configuration/bindings', {
+        templateUrl : 'partials/configuration.bindings.html',
+        controller : 'BindingController',
+        title : 'Configuration'
+    }).when('/configuration/bindings/:bindingId', {
+        templateUrl : 'partials/configuration.bindings.detail.html',
+        controller : 'BindingDetailController',
+        title : 'Configuration'
+    });
+} ]).controller('BindingController', function($scope, $location, bindingRepository) {
+    /**
+     * This is the main binding controller to display all bindings
+     */
+    $scope.navigateTo = function(path) {
+        $location.path(path);
+    }
+
+    $scope.setSubtitle([ 'Bindings' ]);
+    $scope.setHeaderText('Shows all installed bindings.');
+    $scope.refresh = function() {
+        bindingRepository.getAll(true);
+    };
+
+    $scope.configure = function(bindingId, configDescriptionURI, event) {
+        $mdDialog.show({
+            controller : 'ConfigureBindingDialogController',
+            templateUrl : 'partials/dialog.configurebinding.html',
+            targetEvent : event,
+            hasBackdrop : true,
+            locals : {
+                bindingId : bindingId,
+                configDescriptionURI : configDescriptionURI
+            }
+        });
+    }
+    bindingRepository.getAll();
+}).controller('BindingDetailController', function($scope, $location, thingTypeRepository, bindingRepository) {
+    /**
+     * This is the binding controller to display the binding detail
+     */
+    $scope.navigateTo = function(path) {
+        $location.path(path);
+    }
+
+    // Function to allow filtering of thing-types linked to the search box
+    $scope.filterItems = function(lookupFields, searchText) {
+        return function(item) {
+            if (searchText && searchText.length > 0) {
+                for (var i = 0; i < lookupFields.length; i++) {
+                    if (item[lookupFields[i]] && item[lookupFields[i]].toUpperCase().indexOf(searchText.toUpperCase()) != -1) {
+                        return true;
+                    }
+                }
+                return false
+            }
+            return true;
+        }
+    }
+
+    var bindingId = $scope.path[3];
+
+    $scope.searchText = "";
```

Comment:
@htreu I get your arguments.
Currently there is no rule about single vs. double quotes.
If we agree that single quotes should be used for JS we need to change every usage of double quotes in the current JS code base and create a separate PR.

So my suggestion:
* leave it to the author to use single or double as long as there is no rule
* create a issue or a mail on the dev list to discuss if we should add a section to our rules
* ...

WDYT?